deployment/aws-terraform/account-scanner-stack/main.tf

```diff
@@ -0,0 +1,23 @@
+resource "aws_cloudformation_stack" "fss-account-scanner-by-tf" {
+  name = "fss-account-scanner-by-tf"
+  capabilities = ["CAPABILITY_IAM"]
+
+  parameters = {
+    KMSKeyARNForDLQSSE = var.KMSKeyARNForDLQSSE
+    KMSKeyARNForTopicSSE = var.KMSKeyARNForTopicSSE
+    KMSKeyARNsForBucketSSE = var.KMSKeyARNsForBucketSSE
+    ReportObjectKey = var.ReportObjectKey
+    ObjectCreatedEventFilter = var.ObjectCreatedEventFilter
+    ScannerEphemeralStorage = var.ScannerEphemeralStorage
+    QuarantineBucket = var.QuarantineBucket
+    FSSBucketName = var.FSSBucketName
+    FSSKeyPrefix = var.FSSKeyPrefix
+    TrendMicroManagementAccount = var.TrendMicroManagementAccount
+    CloudOneRegion = var.CloudOneRegion
+    ExternalID = var.ExternalID
+    EnableCrossAccountScanning = var.EnableCrossAccountScanning
+  }
+
+  template_url="https://file-storage-security.s3.amazonaws.com/latest/templates/FSS-Account-Scanner-Stack.template"
```

Comment:
For more flexible, you could add a variable such as `TemplateURL` in variables.tf and set a default value `https://file-storage-security.s3.amazonaws.com/latest/templates/FSS-Account-Scanner-Stack.template`.

Reply:
Make sense!